webapp/app.py

```diff
@@ -15,3 +21,13 @@
 @app.route("/")
 def index():
     return render_template("index.html")
+
+
+@app.route("/latest-release.json")
+def latest_release_json():
+    return json.dumps(
+        {
+            "html_url": "https://github.com/CanonicalLtd/multipass/releases/tag/v0.9.0",
+            "tag_name": "v0.9.0"
```

Comment:
@nottrobin we'd need the additional data we had in the YAML :)

```suggestion
            "tag_name": "v0.9.0",
            "title": "Multipass 0.9.0 release",
            "description": "Better desktop integration of the system menu, and more.",
            "download_url": "https://multipass.run/#install",
            "installer_urls": {
              "windows": "https://github.com/CanonicalLtd/multipass/releases/download/v0.9.0/multipass-0.9.0%2Bwin-win64.exe",
              "macos": "https://github.com/CanonicalLtd/multipass/releases/download/v0.9.0/multipass-0.9.0%2Bmac-Darwin.pkg"
            }
```

Reply:
All that said, sure this shouldn't be a static file to avoid the processing overhead?

Reply:
I don't think there's really any processing overhead to speak of. But that is a good point - what sort of traffic can we expect from this?

Sorry I clearly misunderstood your previous comment. I'll put the other keys back in.

Reply:
Okay, I'm flip-flopping on this horribly, but I think I'll go back to `/static/latest-release.json` then. Since this file is going to be longer, that'll make it simpler for you to update, and I think @anthonydillon accepted the argument about `/static` being an okay URL.